### ISSUE TITLE
https://github.com/microsoft/BotFramework-WebChat/issues/5107

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5108]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader/cognitive users will not be aware of which dialog is displayed or might lead to confusion if screen reader does not announce the dialog name.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

Request Contributors - POC (sharepoint.com)

### How to reproduce the issue?

Step 1: Open the above URL in New Edge browsers with valid credentials.
Step 2: Invoke the 'Ask iris now' button which is present under the questions section.->dialog will appear.
Step 3: Verify whether NVDA is not narrating dialog name once 'Iris' dialog gets appeared or not.

### What do you expect?

Once Iris dialog is displayed, the screen reader should announce dialog name & role.
EX: Screen reader should narrate as 'Iris' dialog Menu button.

### What actually happened?

NVDA is not narrating dialog name once 'Iris' dialog gets appeared.

Observation: Screen reader narrating as 'dialog document Menu button  
button Menu Iris button Start Over button Close'

### Do you have any screenshots or recordings to repro the issue?

<img width="958" alt="MAS 4 1 2_NVDA is not narrating dialog name once 'Iris' dialog gets appeared" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/c39f4b0b-7a3c-435b-90ec-9f9c93669f1a">

### MAS reference

https://aka.ms/MAS4.1.2

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS4.1.2;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
Bug copied from MCAPS: [ISSUE 1767706](https://dev.azure.com/worldwidelearning/WWL%20Accessibility/_workitems/edit/1767706)

**Comment by compulim** [comment]
The dialog is not part of Web Chat. This is done by Iris Bot team.

**Comment by Kirank915** [comment]
As per the above comment, this issue is not under bot framework team. Hence closed the bug and updated it to partner team (WWL).
